lib/resources/example_json/r4_examples_coverage.rb

```diff
@@ -226,5 +226,60 @@ module Resources
         }
       ]
     }
+
+    R4_COVERAGE_ENCOUNTER_PATCH ||= [
+      {
+        "op": "replace",
+        "path": "/class/1/name",
+        "value": "name"
+      },
+      {
+        "op": "replace",
+        "path": "/class/0/value",
+        "value": "729825"
+      },
+      {
+        "op": "replace",
+        "path": "/class/1/value",
+        "value": "456"
+      },
+      {
+        "op": "replace",
+        "path": "/dependent",
+        "value": "1626016"
```

Comment:
Can we change this to something that isn't the same as the patient id?

Reply:
https://github.com/cerner/fhir.cerner.com/pull/342/commits/b31e12c97d001c4062999cecf9fa9db4b84409e8#diff-398829d7a4d0e757ca54b5f56e50d99aR249